test/spec/directives/registrationTypeSelectVisibleTypes.spec.js

```diff
@@ -53,19 +53,6 @@ describe('Directive: registrationTypeSelect visibleRegistrantTypes', function()
     expect(typeNames).toContain('Group 1 Dependant 2');
   });
 
-  it('when Group 1 selected on the first screen, only associated registrant types visible on review screen', function() {
```

Comment:
What was the reason to remove this test?

Reply:
Duplicate test, the one right above it is the same one.